src/DssSpell.sol

```diff
@@ -59,167 +55,123 @@ contract DssSpellAction is DssAction {
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
     //
-    uint256 constant ZERO_PCT           = 1000000000000000000000000000;
-    uint256 constant ONE_PCT            = 1000000000315522921573372069;
-    uint256 constant TWO_PCT            = 1000000000627937192491029810;
-    uint256 constant THREE_PCT          = 1000000000937303470807876289;
-    uint256 constant THREE_PT_FIVE_PCT  = 1000000001090862085746321732;
-    uint256 constant FOUR_PCT           = 1000000001243680656318820312;
-    uint256 constant FOUR_PT_FIVE_PCT   = 1000000001395766281313196627;
-    uint256 constant FIVE_PCT           = 1000000001547125957863212448;
-    uint256 constant TEN_PCT            = 1000000003022265980097387650;
+    // uint256 constant ZERO_PCT           = 1000000000000000000000000000;
+    // uint256 constant ONE_PCT            = 1000000000315522921573372069;
+    // uint256 constant TWO_PCT            = 1000000000627937192491029810;
+    // uint256 constant THREE_PCT          = 1000000000937303470807876289;
+    // uint256 constant THREE_PT_FIVE_PCT  = 1000000001090862085746321732;
+    // uint256 constant FOUR_PCT           = 1000000001243680656318820312;
+    // uint256 constant FOUR_PT_FIVE_PCT   = 1000000001395766281313196627;
+    // uint256 constant FIVE_PCT           = 1000000001547125957863212448;
+    // uint256 constant TEN_PCT            = 1000000003022265980097387650;
+
+
+    // Amendment Proposals
+
+    // MIP4c2-SP7: MIP4 Amendments
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/mips/15c887ab8aa761c1008261281669f5cba4bc9803/MIP4/MIP4c2-Subproposals/MIP4c2-SP7.md -q -O - 2> /dev/null)"
```

Comment:
I would check the commit in the urls as they point out to the last commit on master branch.
It should be the latest commit per mip as follow:
```suggestion
    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/mips/a4ec795c3e088de3def2312ed4c5df9c3b0fb192/MIP4/MIP4c2-Subproposals/MIP4c2-SP7.md -q -O - 2> /dev/null)"
```

Reply:
done! please help me review that

Reply:
:heavy_check_mark: Commits Matches Last Commit per MIP